packages/blockchain-wallet-v4-frontend/src/data/components/buySell/sagas.ts

```diff
@@ -1122,10 +1170,17 @@ export default ({ api, coreSagas, networks }: { api: APIType; coreSagas: any; ne
         const refresh = -moment().add(10, 'seconds').diff(quote.quoteExpiresAt)
         yield delay(refresh)
       } catch (e) {
-        const error = errorHandler(e)
+        const { code: network_error_code, message: network_error_description } =
+          errorCodeAndMessage(e)
+        const error: PartialClientErrorProperties = {
+          network_endpoint: '/brokerage/quote',
+          network_error_code,
+          network_error_description,
+          source: 'NABU'
+        }
         yield put(A.fetchBuyQuoteFailure(error))
-        yield delay(FALLBACK_DELAY)
-        yield put(A.startPollBuyQuote(payload))
+        // stop fetching new quote until user does retry action
+        yield put(A.stopPollBuyQuote())
```

Comment:
This change isn't directly related the rest of the changes in this PR but I noticed that we keep hitting the quotes endpoint here even though the user has been transitioned to the Oops error screen already so we're just making these quote calls in the background for no reason.